bin/generate-stories.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
+
+const dextroseExtension = '.dextrose.tmp.js';
+
+const overRideStories = (dirPath, files, stringToFind, stringToReplace) =>
+  new Promise((resolve, reject) => {
+    files.forEach((file) => {
+      fs.readFile(file, 'utf8', (err, data) => {
+        if (err) {
+          reject(err);
+        }
+        if (data.includes(stringToFind)) {
+          const updatedData = data.replace(stringToFind, stringToReplace);
+          const trimmedFile = file.replace('.js', '');
+          const newFilePath = trimmedFile.concat(dextroseExtension);
+          fs.writeFileSync(`${dirPath}/${newFilePath}`, updatedData);
+        }
+      });
+    });
+    resolve();
+  });
+
+const findStoryFiles = (dir, matchers) => new Promise((resolve, reject) => {
+  fs.readdir(dir, 'utf8', (err, files) => {
+    if (err) reject(err);
```

Comment:
Just to check, I think @toboid said we shouldn't be using one line if statements? Does this not apply on the dextrose app?

Reply:
dextrose is new and cool we use this a couple times throughout